catalogs/data/airbyte/terraform/azure/outputs.tf

```diff
@@ -0,0 +1,28 @@
+output "connection_string" {
+  value = data.azurerm_storage_account.airbyte.primary_connection_string
+  sensitive = true
+}
+
+output "postgres_host" {
+  value = azurerm_postgresql_flexible_server.postgres.fqdn
+}
+
+output "postgres_password" {
+  value = azurerm_postgresql_flexible_server.postgres.administrator_password
```

Comment:
I wouldn't be surprised if this is actually goofy somehow, could you use the output from the `random_password.db_password` instead

Reply:
I have checked it via state and outputs and both were the same when I did. I will update it anyways.

Reply:
It's done.